examples/tina-self-hosted-demo/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.49",
   "private": true,
   "scripts": {
-    "dev": "TINA_PUBLIC_IS_LOCAL=true tinacms dev -c \"next dev\"",
+    "dev": "tinacms dev -c \"next dev\"",
```

Comment:
Just to double check - do we need to update the docs or add this to a `.nev` file `TINA_PUBLIC_IS_LOCAL=true`
I believe this might still be needed

Reply:
the better approach will be remove this change and update the docs, so that window user will not have issue when running the self hosted demo

Changes reverted